examples/graphics/render-to-texture.html

```diff
@@ -69,86 +77,112 @@
 
-        // get world layer
+        // get world and skybox layers
         var worldLayer = app.scene.layers.getLayerByName("World");
+        var skyboxLayer = app.scene.layers.getLayerByName("Skybox");
 
-        // create ground plane and 3 primitives, visible in both layers
-        createPrimitive("plane", new pc.Vec3(0, 0, 0), new pc.Vec3(20, 20, 20), new pc.Color(0.3, 0.5, 0.3), [worldLayer.id, tvLayer.id]);
-        createPrimitive("sphere", new pc.Vec3(-2, 1, 0), new pc.Vec3(2, 2, 2), new pc.Color(1, 0, 0), [worldLayer.id, tvLayer.id]);
-        createPrimitive("box", new pc.Vec3(2, 1, 0), new pc.Vec3(2, 2, 2), new pc.Color(1, 1, 0), [worldLayer.id, tvLayer.id]);
-        createPrimitive("cone", new pc.Vec3(0, 1, -2), new pc.Vec3(2, 2, 2), new pc.Color(0, 1, 1), [worldLayer.id, tvLayer.id]);
+        // create ground plane and 3 primitives, visible in world layer
+        createPrimitive("plane", new pc.Vec3(0, 0, 0), new pc.Vec3(20, 20, 20), new pc.Color(0.2, 0.4, 0.2), [worldLayer.id]);
+        createPrimitive("sphere", new pc.Vec3(-2, 1, 0), new pc.Vec3(2, 2, 2), pc.Color.RED, [worldLayer.id]);
+        createPrimitive("box", new pc.Vec3(2, 1, 0), new pc.Vec3(2, 2, 2), pc.Color.YELLOW, [worldLayer.id]);
+        createPrimitive("cone", new pc.Vec3(0, 1, -2), new pc.Vec3(2, 2, 2), pc.Color.CYAN, [worldLayer.id]);
 
-        // Create main camera, which renders entities in world layer
-        var camera = new pc.Entity();
+        // Create main camera, which renders entities in world, excluded and skybox layers
+        var camera = new pc.Entity("Camera");
         camera.addComponent("camera", {
-            clearColor: new pc.Color(0.2, 0.2, 0.4),
-            layers: [worldLayer.id]
+            fov: 100,
+            layers: [worldLayer.id, excludedLayer.id, skyboxLayer.id]
         });
-        camera.translate(0, 20, 25);
-        camera.lookAt(pc.Vec3.ZERO);
+        camera.translate(0, 9, 15);
+        camera.lookAt(1, 4, 0);
         app.root.addChild(camera);
 
-        // Create tv camera, which renders entities in tv layer
-        var tvCamera = new pc.Entity();
-        tvCamera.addComponent("camera", {
-            clearColor: new pc.Color(0.2, 0.2, 0.4),
-            layers: [tvLayer.id]
+        // Create texture camera, which renders entities in world and skybox layers into the texture
+        var textureCamera = new pc.Entity("TextureCamera");
+        textureCamera.addComponent("camera", {
+            layers: [worldLayer.id, skyboxLayer.id],
+
+            // set the priority of textureCamera to lower number than the priority of the main camera (which is at default 0)
+            // to make it rendered first each frame
+            priority: -1
         });
-        // add sphere at the position of tv camera to see it in the world
-        tvCamera.addComponent("model", {
+
+        // this camera renders into texture target
+        textureCamera.camera.renderTarget = renderTarget;
```

Comment:
Can this be supplied in the `addComponent` options?

Reply:
done, thanks.